modules/nw-ovn-kubernetes-running-ovnkube-trace.adoc

```diff
@@ -200,21 +210,37 @@ $ ./ovnkube-trace \
  -loglevel 2
 ----
 +
-.Expected output
-
+.Example output
 [source,terminal]
 ----
+I0915 15:22:39.161246   97445 ovnkube-trace.go:1160] Log level set to: 2
+I0915 15:22:47.063131   97445 ovnkube-trace.go:696] ovn-trace source pod to destination pod Output:
+# tcp,reg14=0xf,vlan_tci=0x0000,dl_src=0a:58:0a:83:00:10,dl_dst=0a:58:0a:83:00:01,nw_src=10.131.0.16,nw_dst=10.131.0.15,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=52888,tp_dst=80,tcp_flags=0
+
+ingress(dp="ci-ln-gwvv70k-72292-kk5cf-worker-c-547z2", inport="prod_test-6459")
+-------------------------------------------------------------------------------
+ 0. ls_in_check_port_sec (northd.c:9114): 1, priority 50, uuid 65ab31e0
+    reg0[15] = check_in_port_sec();
+    next;
+ 4. ls_in_pre_acl (northd.c:6606): ip, priority 100, uuid a4cf6d66
+    reg0[0] = 1;
+    next;
+ 5. ls_in_pre_lb (northd.c:6793): ip, priority 100, uuid a4e6535f
+    reg0[2] = 1;
+    next;
+ 6. ls_in_pre_stateful (northd.c:6816): reg0[2] == 1, priority 110, uuid f5809fa5
+    ct_lb_mark;
+
 ct_lb_mark /* default (use --ct to customize) */
 ------------------------------------------------
- 3. ls_out_acl_hint (northd.c:6092): !ct.new && ct.est && !ct.rpl && ct_mark.blocked == 0, priority 4, uuid 32d45ad4
+ 7. ls_in_acl_hint (northd.c:6912): !ct.new && ct.est && !ct.rpl && ct_mark.blocked == 0, priority 4, uuid c16f415c <1>
```

Comment:
seems the output cannot indicate the reason why traffic is blocked, please check if there any output info about ingressDefaultDeny, i think it's better to show that here ^ ^